neptune-client-api.md

```diff
@@ -0,0 +1,56 @@
+
+
+```python
+from neptune.session import Session
+
+session = Session("token=")
```

Comment:
@lukasz-walkiewicz I think, from the user perspective if we could join the session/project/exp into a single constructor it would gain some extra usability points. 

I see the point though. With this I can query this project for things (like in the neptune-lib now) so I see why we need this object. 

How do you feel about having an ugly wrapper function:

```python
exp = create_experiment(token=,
                                         project=,
                                         params=,
                                         ...
                                         )
```

that under the hood contructs session and project objects?

Reply:
We've changed that part recently. Would you please take another look?